protokube/pkg/protokube/do_volume.go

```diff
@@ -128,9 +134,33 @@ func NewDOVolumes() (*DOVolumes, error) {
 	}, nil
 }
 
+// Token() returns oauth2.Token
+func (t *TokenSource) Token() (*oauth2.Token, error) {
+	token := &oauth2.Token{
+		AccessToken: t.AccessToken,
+	}
+	return token, nil
+}
+
+func NewDOCloud() (*godo.Client, error) {
+	accessToken := os.Getenv("DIGITALOCEAN_ACCESS_TOKEN")
+	if accessToken == "" {
+		return nil, errors.New("DIGITALOCEAN_ACCESS_TOKEN is required")
+	}
```

Comment:
This looks like a functional modification. I can't see any corresponding deletion.

Reply:
Protokube had a dependency on resources - digital ocean package and was getting a newCloud object by calling the dependency [here](https://github.com/kubernetes/kops/pull/11592/files?file-filters%5B%5D=.go#diff-c8f5df1bde4e360ef9070eb81ce136cc4d3c99220c0dc157300898b002826967L106)
Now, since that dependency is removed, it directly depends on the godo client and creates a new object here.

Reply:
> Unfortunately, the second commit is to big. I cannot easily follow to verify the lack of functional modifications. The fact that functions are being reordered at the same time as they or other things are being modified makes the diffs extremely hard to follow.
> 
> It also looks like the first commit won't compile, but is fixed up in the second commit. I can work around some of that, but it makes reviewing a harder task.

@johngmyers - Apologize, its creating a problem in reviewing this. My first commit is failing since I just moved the same file over to upup/pkg/fi/cloudup/do. But I fixed all issues in my second commit, but placing of the functions didn't match well with the original code.

Reply:
I can see the corresponding deletion now. Thanks.